aion/config/dev.exs

```diff
@@ -26,18 +19,21 @@ config :aion, Aion.Endpoint,
     ]
   ]
 
-# Do not include metadata nor timestamps in development logs
 config :logger, :console, format: "[$level] $message\n"
 
-# Set a higher stacktrace during development. Avoid configuring such
-# in production as building large stacktraces may be expensive.
 config :phoenix, :stacktrace_depth, 20
 
-# Configure your database
 config :aion, Aion.Repo,
   adapter: Ecto.Adapters.Postgres,
   username: "postgres",
   password: "postgres",
   database: "aion",
   hostname: "localhost",
   pool_size: 10
+
+config :guardian, Guardian,
+  allowed_algos: ["HS512"],
+  secret_key: "@11H41LW35tT3x45",
```

Comment:
:laughing:

Reply:
Thats super experimental PR, please do not laugh 😠